src/adapter/src/catalog/storage.rs

```diff
@@ -250,32 +247,6 @@ async fn migrate(
                     name: "information_schema".into(),
                 },
             )?;
-            txn.roles.insert(
```

Comment:
Do we need to write a migration that REMOVES this role from existing stashes?

Reply:
I thought about that, however I was worried that there's some extremely small but non-zero chance that some existing deployment is using the `materialize` role, if one of their users are named "materialize". Though perhaps we should still delete it and let if be re-created next time they log in.